src/main/java/uk/ac/bath/se/DungeonController.java

```diff
@@ -286,18 +287,54 @@ public void movePlayer(PlayerMovement move) {
     }
 
     private void moveBot() {
-        //TODO refactor
-        int[] randomWalk = BotPlayer.randomWalk(dungeon);
-        while (gridBounds[randomWalk[0]][randomWalk[1]] != 0) {
-            randomWalk = BotPlayer.randomWalk(dungeon);
+        PlayerMovement botMove;
+        if (Math.abs(botPlayer.getXCoord()-player.getPlayerX()) >= Math.abs(botPlayer.getYCoord()-player.getPlayerY())) {
```

Comment:
Why math.abs? They're integers anyway. Is the desired behaviour to not be able to have negative numbers?